spec/controllers/notifications_controller_spec.rb

```diff
@@ -31,6 +31,19 @@
       get :update, "id" => note.id, :set_unread => "true", :format => :json
     end
 
+    it 'marks a notification as unread without timestamping' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.